starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  font-family: ‘Source Sans Pro’, sans-serif;
+  font-weight: 100;
+  font-size: 14px;
+  color: #333;
+  width: 100%;
+  height: 100%;
+}
+
+header {
+  background-image: url("../images/apple-pie.jpg");
+  width: 100%;
+  height: 60%;
+  display: flex;
+}
+
+h1 {
+    font-size: 48px;
+    font-weight: bolder;
+}
+
+h2 {
+    font-size: 24px;
+    font-weight: lighter;
+}
+
```

Comment:
I would try to avoid using h1 or h2 as much as possible, as it will better practice to use classes instead.